dsl/read.go

```diff
@@ -103,7 +103,7 @@ func ReadHopsFileContents(hopsFileContent []FileContent) (*hcl.BodyContent, stri
 	}
 
 	if len(content.Blocks) == 0 {
-		return nil, "", errors.New("At least one resource must be defined in your hops config(s)")
+		return nil, "", errors.New("Ensure the hops folder path is correctly defined. The hops folder should contain at least one resource")
```

Comment:
This is suprisingly tricky!

The relevant documentation for this is [here](https://docs.hiphops.io/#/hops-configuring) at "Config location". The rule is a bit fiddly. Basically there are two things we are looking for:

- That there is at least 1 hops file with pipelines/tasks defined in it (not sure what the right terminology is)
- And that hops file is in an "automation bundle" which is basically a subdirectory under the hops folder path. There can be as many subdirectories as we want.

I'm personally struggling to find a succinct way to phrase all that.

Something like this?

"Ensure the hops folder path is correctly defined and pointing to at least one automation bundle (a subfolder with a .hops file with a resource)"

Maybe we add the link above to the docs as well? (We've not done this before).

Reply:
For now, let's see if we can do it without the link to the docs, I think.

Reply:
"No automations found. Ensure the hops folder path is correctly defined and pointing to at least one automation bundle (a subfolder with a .hops file with a resource)" might do the trick.